src/components/Input/Input.tsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import classNames from 'classnames';
-import { ClassValue } from 'classnames/types';
 
 import styles from './Input.module.css';
 
-interface InputProps {
-  className?: ClassValue;
+interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {
   /** Applies disabled appearance without disabling. Helpful for adding tooltips on disabled field clicks. */
   fauxDisabled?: boolean;
 }
 
 const Input: React.FC<InputProps> = ({
   className = '',
+  onChange = () => {},
```

Comment:
Do you need to add onChange and readOnly? I think they should be handled fine by passedProps.

Reply:
This was actually due to some suggestions from React warnings. This was what it said:

> checkPropTypes.js:20 Warning: Failed prop type: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.

I didn't know `defaultValue` existed, but it seemed reasonable to provide defaults based on this.

Reply:
Also, thank you for asking. I completely forgot to leave a comment clarifying the addition

Reply:
Looks good, thanks for clarifying!